Infrastructure/Resources/New-StorageAccountContainer.ps1

```diff
@@ -30,7 +30,11 @@ Param(
     [Parameter(Mandatory = $true)]
     [String]$Name,
     [Parameter(Mandatory = $true)]
-    [String[]]$ContainerName
+    [String[]]$ContainerName,
+    [Parameter(Mandatory= $true)]
+    [ValidateSet("Container", "Blob", "Off")]
+    [String[]]$ContainerPermission
```

Comment:
```suggestion
    [Parameter(Mandatory= $false)]
    [ValidateSet("Container", "Blob", "Off")]
    [String]$ContainerPermission = "Off"
```

-  Mandatory= $false
-  [String] rather than [String[]]
-  Default value = "Off"

Reply:
done